x/bounty/client/cli/tx.go

```diff
@@ -317,3 +318,83 @@ func HostProcessFinding(cmd *cobra.Command, args []string) (fid uint64,
 
 	return fid, commentAny, hostAddr, nil
 }
+
+func NewReleaseFindingCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "release-finding",
```

Comment:
Use `release-finding 1 ` instead of `release-finding --finding-id 1` , it may be more user-friendly

Reply:
I think all commands should use the same command format

Reply:
Yes, there are currently 2 formats, and should be unified.